online-banking-cz/src/main/java/com/adyen/checkout/onlinebankingcz/OnlineBankingCZView.kt

```diff
@@ -122,7 +122,7 @@ class OnlineBankingCZView @JvmOverloads constructor(
 
         val spannable = SpannableString(newString).apply {
             setSpan(
-                ForegroundColorSpan(resources.getColor(R.color.textColorLink, null)),
+                ForegroundColorSpan(resources.getColor(R.color.textColorLink)),
```

Comment:
Isn't this method deprecated?

Reply:
it's, but I got an error because we're using min api level 21 and it's deprecated in api level 23 I think not sure though when it was deprecated

Reply:
You can use `ResourcesCompat.getColor`